### PR TITLE
Chapter 10: Manage User

### PR DESCRIPTION
**Pull request by ainulfarizah**
ready

### CODE REVIEW

**Comment by ainulfarizah** [comment]
ready

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<% provide(:title, t(:all_users)) %>
+
+<h1>All users</h1>
+
+<%= will_paginate %>
+
+<ul class="users">
+    <%= render @user %>
+  <% end %>
```
Comment (anlvq-1058) [inline app/views/users/index.html.erb:9]: end for what ?

**Review by ghost — COMMENTED** [review]
cmt

**Comment by ainulfarizah** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**